int64[] num_vector # Request
int64 a
int64 b
---
int64 cumulative_sum  # Response
